scripts/listDatabase.sh

```diff
@@ -25,7 +35,12 @@ then
     exit
 fi
 
-keys=($(get_keys))
+if [ ! -f "$cacheFile" ] || [ $interval -gt 60 ];then
+	# update cache file
+	echo "$(get_keys)" > "$cacheFile"
+fi
+# get keys from cache 
+keys=($(cat $cacheFile | grep -i "{query}" ))
```

Comment:
Will this cache only available query keys from the database or also password values?
This will also only work after entry was searched at least once, would it be better to prepare the cache once beforehand, caching all the entries without passwords?

Reply:
1. No password will be cached.
2. All entries will be cached every time refreshing the cache file.